src/main/java/com/moneylendingapp/security/UsersDetails.java

```diff
@@ -0,0 +1,62 @@
+package com.moneylendingapp.security;
+
+import com.moneylendingapp.entities.User;
+import lombok.AllArgsConstructor;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+@AllArgsConstructor
+public class UsersDetails implements UserDetails {
+
+    private final String username;
+    private String password;
+    private List<GrantedAuthority> authorities;
+
+
+    public UsersDetails(User user) {
+        this.username = user.getUsername();
+        this.password = user.getPassword();
```

Comment:
Please don't do this. let the password field be empty or null.
Don't load your password into the memory after login